internal/controller/core/device_controller.go

```diff
@@ -133,29 +126,46 @@ func (r *DeviceReconciler) Reconcile(ctx context.Context, req ctrl.Request) (_ c
 		})
 		obj.Status.Phase = v1alpha1.DevicePhaseProvisioning
 		r.Recorder.Event(obj, "Normal", "ProvisioningStarted", "Device provisioning has started")
-		// TODO(swagner-de): Start POAP Process.
-		_ = tmpl // <-- Use the template.
 		return ctrl.Result{RequeueAfter: r.RequeueInterval}, nil
```

Comment:
```suggestion
		return ctrl.Result{}, nil
```
Can we remove this, so we immediately start the provisioning?